doc/modules/pipeline.rst

```diff
@@ -148,7 +149,7 @@ object::
     >>> pipe # doctest: +NORMALIZE_WHITESPACE, +ELLIPSIS
     Pipeline(...,
              steps=[('reduce_dim', PCA(copy=True,...)),
-                    ('clf', SVC(C=1.0,...))])
+                    ('clf', SVC(C=1.0,...))], verbose=False)
```

Comment:
Well then i'll wait for others' suggestion meanwhile. I'm not sure which should be preferred either.